.github/workflows/test.yml

```diff
@@ -18,27 +18,51 @@ on:
 # TODO split tests into mongo related & non mongo related. Run the former once.
 jobs:
 
+  workspace_integration_tests:
```

Comment:
Can we call this workspace_container_tests or something like that to disambiguate from the integration tests in the test suite?

Reply:
:+1: